src/components/diff-view.jsx

```diff
@@ -44,28 +46,44 @@ export default class DiffView extends React.Component {
   }
 
   render () {
-    const { a, b, diffType } = this.props;
-    const { diff } = this.state;
+    const { a, b } = this.props;
+    const chosenDiffType = this.props.diffType;
+    const { diff, loadDiffType } = this.state;
 
-    if (!diffType || !diffTypes[diffType] || !diff) {
-      return null;
+    if (chosenDiffType && diffTypes[chosenDiffType].diff_service === 'TODO') {
+      return <div>No diff for '{diffTypes[chosenDiffType].description}' yet</div>;
+    }
+
+    if (!loadDiffType ||
+        !diff ||
+        loadDiffType !== this.props.diffType)
```

Comment:
See notes about not tracking `loadDiffType`. I think this can just be `if (!this.state.diff)`